packages/venia-ui/i18n/en_US.json

```diff
@@ -108,6 +122,9 @@
     "createAccount.passwordText": "Password",
     "createAccount.subscribeText": "Subscribe to news and updates",
     "createWishlist.createListText": "Create a list",
+    "createWishlist.dialogTitle": "New Favorites List",
+    "createWishlist.handleCreateListText": "Create a list",
+    "createWishlist.listName": "List Name",
```

Comment:
Where did these come from?

Reply:
There were console warnings complaining about missing translations, so they've been added. Traced it back to a feature merge ~6 days ago that did not have these included.